source/stylesheets/refills/_search-tools.scss

```diff
@@ -61,7 +65,7 @@
       border: 1px solid $base-border-color;
       font-size: 1em;
       margin-bottom: $base-spacing / 2;
-      padding: ($base-spacing / 3) $clear-button-width ($base-spacing / 3) ($base-spacing / 3);
+      padding: ($base-spacing / 3) ($base-spacing / 3);
```

Comment:
Shorthand form for property `padding` should be written more concisely as `$base-spacing / 3` instead of `$base-spacing / 3 $base-spacing / 3`